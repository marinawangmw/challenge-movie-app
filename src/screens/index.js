export { Home } from '@/screens/Home/Home';
export { Login } from '@/screens/Login/Login';
export { Profile } from '@/screens/Profile/Profile';
export { Loading } from '@/screens/Loading/Loading';
export { MovieCollection } from '@/screens/MovieCollection/MovieCollection';
export { MovieDetail } from '@/screens/MovieDetail/MovieDetail';
