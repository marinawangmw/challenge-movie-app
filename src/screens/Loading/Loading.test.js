import React from 'react';
import renderer from 'react-test-renderer';
import { Loading } from '@/screens';

describe('Loading renders correcly', () => {
  it('should match the snapshot', () => {
    const tree = renderer.create(<Loading />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
