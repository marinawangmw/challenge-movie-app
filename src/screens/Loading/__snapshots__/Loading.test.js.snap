// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders correcly should match the snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flex": 1,
        "height": "110%",
        "justifyContent": "center",
        "position": "absolute",
        "width": "100%",
      },
      Object {
        "backgroundColor": "rgb(242, 242, 242)CC",
      },
    ]
  }
>
  <ActivityIndicator
    animating={true}
    color="#999999"
    hidesWhenStopped={true}
    size="large"
  />
  <Text
    testID="name"
  >
    Hello, undefined!
  </Text>
</View>
`;
